Sources/SwiftFileStore/FileObjectStore.swift

```diff
@@ -70,6 +70,24 @@ public final class FileObjectStore: ObjectStore {
     }
     return try await removeAllTask.value
   }
+    
+    public func readAllKeys(namespace: String) async throws -> [String] {
+        let readAllTask = Task {() -> [String] in
```

Comment:
nit: No need to wrap the code in `Task` as it is already executed within an async function.

Reply:
I do not quite understand, i copy from 'read' function.

Reply:
merge latest changes to `main`.